cmd/juju/status/health.go

```diff
@@ -0,0 +1,252 @@
+// Copyright 2016 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package status
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+
+	"launchpad.net/gnuflag"
+
+	"github.com/juju/cmd"
+	"github.com/juju/errors"
```

Comment:
No big deal I'll fix that but I was just trying to follow the contributing guide which says "Import statements are grouped into 3 sections: standard library, 3rd party libraries, juju imports."